reproducibility_project/spe_subproject/summarize.py

```diff
@@ -0,0 +1,162 @@
+        "tail_energy": "tail_energy",
+        "kspace_energy": "long_range_electrostatics",
+    }
+
+    for job in project.find_jobs({"engine": "lammps-VU"}):
+        data = np.genfromtxt(f"{job.ws}/log-spe.txt", names=True, delimiter=",")
+        for prop in data.dtype.names:
+            spe_data[job.sp.molecule][job.sp.engine][lmp_map[prop]] += data[
+                prop
+            ]
+
+    # Parse HOOMD SPE data
+    hoomd_map = {
+        "pair_LJ": "tot_pair_energy",
+        "pair_LJ_tail": "tail_energy",
+        "pair_Ewald": "short_range_electrostatics",
+        "pppm_Coulomb": "long_range_electrostatics",
+        "special_pair_LJ": "tot_pair_energy",
+        "special_pair_Coulomb": "short_range_electrostatics",
+        "bond_Harmonic": "bonds_energy",
+        "angle_Harmonic": "angles_energy",
+        "dihedral_OPLS": "dihedrals_energy",
+        "potential_energy": "potential_energy",
+    }
+    for job in project.find_jobs({"engine": "hoomd"}):
+        data = np.genfromtxt(
+            f"{job.ws}/log-spe-raw.txt", names=True, delimiter=" "
+        )
+        for prop in data.dtype.names:
+            spe_data[job.sp.molecule][job.sp.engine][hoomd_map[prop]] += data[
+                prop
+            ]
+
+    # Parse GROMACS SPE data
+    for job in project.find_jobs({"engine": "gromacs"}):
+        data = np.genfromtxt(
+            f"{job.ws}/log-spe-p3m.txt", names=True, delimiter=","
+        )
+        for prop in data.dtype.names:
+            if prop in properties:
+                spe_data[job.sp.molecule][job.sp.engine][prop] += data[prop]
+            else:
+                print(f"Not included {job.sp.engine} {job.sp.molecule} {prop}")
+
+    # Parse MCCCS SPE data
+    for job in project.find_jobs({"engine": "mcccs"}):
+        data = np.genfromtxt(f"{job.ws}/log-spe.txt", names=True, delimiter=",")
+        for prop in data.dtype.names:
+            if prop in properties:
+                spe_data[job.sp.molecule][job.sp.engine][prop] += data[prop]
+            else:
+                print(f"Not included {job.sp.engine} {job.sp.molecule} {prop}")
+
+    # Parse Cassandra SPE data
+    for job in project.find_jobs({"engine": "cassandra"}):
+        data = np.genfromtxt(f"{job.ws}/log-spe.txt", names=True, delimiter=",")
+        for prop in data.dtype.names:
+            if prop in properties:
+                spe_data[job.sp.molecule][job.sp.engine][prop] += data[prop]
+            else:
+                print(f"Not included {job.sp.engine} {job.sp.molecule} {prop}")
+
+    # Parse GOMC SPE data
+    gomc_map = {
+        "TOTAL": "potential_energy",
+        "INTRA(B)": "intramolecular_energy",
+        "INTRA(NB)": "intramolecular_energy",
+        "BOND(B)": "bonds_energy",
+        "ANGLE(B)": "angles_energy",
+        "DIHEDRAL(B)": "dihedrals_energy",
+        "INTER(LJ)": "intermolecular_energy",
+        "LRC": "tail_energy",
+        "TOTAL_ELECT": "total_electrostatics",
```

Comment:
I believe `total_electrostatics`, should be `intermolecular_total_electrostatics`, because the INTRA(NB) part covers the `intramolecular_total_electrostatics`